### PR TITLE
Add many entry for cheminformatics

### PR DESCRIPTION
**Pull request by Birdlet**
Add many entry for databases, docking, md, and issues in your repo.
I will provide more usefull tools in future

### CODE REVIEW

**Comment by hsiaoyi0504** [comment]
@Birdlet I am glad that you want to contribute. However, I would like to keep commits clearer. Therefore,

1. Try to keep only one major update in each pull request
2. Keep commit message clear

File: README.md [hunk 1]
```diff
@@ -47,6 +51,7 @@ A curated list of awesome Cheminformatics software, resources, and libraries. Mo
 
 * [AutoDock Vina](http://vina.scripps.edu/) - Molecular docking and virtual screening.
 * [smina](https://sourceforge.net/projects/smina/) - Customized [AutoDock Vina](http://vina.scripps.edu/) to better support scoring function development and high-performance energy minimization.
+* [rDock]
```
Comment (hsiaoyi0504) [inline README.md]: This doesn't have a URL.

File: README.md [hunk 2]
```diff
@@ -162,6 +162,8 @@ __tags:__
 * [PDBFixer](https://github.com/pandegroup/pdbfixer) - PDBFixer is an easy to use application for fixing problems in Protein Data Bank files in preparation for simulating them. :book:
 * [Biopython](https://biopython.org/) - Biopython is a set of freely available tools for biological computation written in Python by an international team of developers. :book:
 * [ODDT](https://github.com/oddt/oddt) - Open Drug Discovery Toolkit (ODDT) is modular and comprehensive toolkit for use in cheminformatics, molecular modeling etc. ODDT is written in Python, and make extensive use of Numpy/Scipy. :book:
+* [ProDy](http://prody.csb.pitt.edu) - ProDy is a free and open-source Python package for protein structural dynamics analysis, especially Elastic Network Model. :book:
```
Comment (hsiaoyi0504) [inline README.md]: The commit message is meaningless ...

**Comment by hsiaoyi0504** [comment]
For example, 
1. add emoji in one pull request
2. add multiple entries in the same category in one pull request. If there are more than one categories you want to update, include one and only one in per pull request.

File: README.md [hunk 3]
```diff
@@ -4,21 +4,24 @@
...
+* Free: :credit_card:
+* Comercial: :moneybag:
+
 ## Contents
 
 * [Applications](#applications)
-  * [Visualization](#app-visualization)
-  * [Format Conversion](#app-cmd)
-  * [Docking](#app-docking)
-* [Libraries](#libraries)
```
Comment (hsiaoyi0504) [inline README.md]: I would like to keep categories (applications and libraries) separate.
The major difference is that libraries are software that we can further extend it natively. However, application can't be extended natively.

File: README.md [hunk 4]
```diff
@@ -28,82 +31,147 @@ A curated list of awesome Cheminformatics software, resources, and libraries. Mo
...
+* [Excape](https://sandbox.ideaconsult.net/search/excape) - ExCAPE-DB is a large public chemogenomics dataset based on the PubChem and ChEMBL databases, and large scale standardisation (including tautomerization) of chemical structures using open source cheminformatics software. :book:
+* [ZINC](http://zinc.docking.org/) - ZINC is a free database of commercially-available compounds for virtual screening.   ZINC contains over 35 million purchasable compounds in ready-to-dock, 3D formats.   ZINC is provided by the Irwin and Shoichet Laboratories in the Department of Pharmaceutical Chemistry at the University of California, San Francisco (UCSF). :book:
+* [ZINC15](http://zinc15.docking.org/) - Welcome to ZINC, a free database of commercially-available compounds for virtual screening. ZINC contains over 230 million purchasable compounds in ready-to-dock, 3D formats. ZINC also contains over 750 million purchasable compounds you can search for analogs in under a minute.  :book:
```
Comment (hsiaoyi0504) [inline README.md]: I think ZINC and ZINC 15 can be included in one entry.

File: README.md (hunk 2 above)
Comment (hsiaoyi0504) [reply README.md]: I mean 23fb33ada15dbbf3ffe847bb7a99fbbdd62f61d6 here.

File: README.md [hunk 5]
```diff
@@ -116,6 +184,9 @@ A curated list of awesome Cheminformatics software, resources, and libraries. Mo
 
 * [Learncheminformatics.com](http://learncheminformatics.com/) - "Cheminformatics: Navigating the world of chemical data" courese at Indiana University.
 * [cheminfoeducation](https://www.youtube.com/user/cheminfoeducation/videos) - A YouTube channel for cheminformatics education.
+* [CH485-Artificial-Intelligence-and-Chemistry](https://github.com/Birdlet/CH485---Artificial-Intelligence-and-Chemistry) - Course material for Artificial Intelligence and Chemistry of Korea Advanced Institute of Science and Technology (KAIST).
+[CHEM430-SIMULATION IN CHEMISTRY & BIOCHEMISTRY](https://dasher.wustl.edu/chem430/) - IMULATION IN CHEMISTRY & BIOCHEMISTRY by Department of Chemistry at Washington University.
+[CHEM478-MOLECULAR MODELING](https://dasher.wustl.edu/chem430/) - MOLECULAR MODELING by Department of Chemistry at Washington University.
```
Comment (hsiaoyi0504) [inline README.md]: I like these courses. Thanks for contribution.

File: README.md [hunk 6]
```diff
@@ -132,11 +203,13 @@ A curated list of awesome Cheminformatics software, resources, and libraries. Mo
...
 * [steeveslab-blog](http://asteeves.github.io/) - Some examples using [RDKit](http://www.rdkit.org/).
 * [Macs in Chemistry](http://www.macinchem.org/) - Provide a resource for chemists using Apple Macintosh computers.
+* [Is Life Worth Living](https://iwatobipen.wordpress.com) - iwatobipen is a medicinal chemist in mid-size pharmaceutical company in Japan. He loves cheminformatics and posted lots of chemoinformatical blogs
 
 ### Books
 
 * [Computational Approaches in Cheminformatics and Bioinformatics](https://books.google.com/books/about/Computational_Approaches_in_Cheminformat.html?id=bLqV4rYQoYsC) -  Include insights from public (NIH), academic, and industrial sources at the same time.
 * [Chemoinformatics for Drug Discovery](https://onlinelibrary.wiley.com/doi/book/10.1002/9781118742785) - Materials about how to use Chemoinformatics strategies to improve drug discovery results.
+* [Handbook of Chemoinformatics: From Data to Knowledge in 4 Volumes](https://onlinelibrary.wiley.com/doi/book/10.1002/9783527618279) - Many detials for cheminformatics descriptors and algorithm. RDkit cited this book alot.
```
Comment (hsiaoyi0504) [inline README.md]: A typo here. There should be a space between a and lot.

File: README.md [hunk 7]
```diff
@@ -28,82 +31,147 @@ A curated list of awesome Cheminformatics software, resources, and libraries. Mo
...
 
-<a id="lib-db"></a>
 ### Databases
 
-* [razi](https://github.com/rvianello/razi) - Cheminformatic extension for the SQLAlchemy database.
+#### Databases
+
+* [PubChem](https://pubchem.ncbi.nlm.nih.gov/) - PubChem is world's largest collection of freely accessible chemical information. Search chemicals by name, molecular formula, structure, and other identifiers. Find chemical and physical properties, biological activities, safety and toxicity information, patents, literature citations and more. :book:
+* [ChEMBL](https://www.ebi.ac.uk/chembl/) - ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs. :book:
+* [Excape](https://sandbox.ideaconsult.net/search/excape) - ExCAPE-DB is a large public chemogenomics dataset based on the PubChem and ChEMBL databases, and large scale standardisation (including tautomerization) of chemical structures using open source cheminformatics software. :book:
```
Comment (hsiaoyi0504) [inline README.md]: The one is weird. It looks like that this resource is private.

Comment (Birdlet) [reply README.md]: sorry, this one is excape https://solr.ideaconsult.net/search/excape/

**Comment by Birdlet** [comment]
Thank you @hsiaoyi0504 , I really appreciate your advice for what I write. I will carefully follow your suggestion to fix these proplem. I think I should close this pull request and start from a new brunch... 

Thank you again!

**Pull request closed without merging**